Prepare testing in tile mode for libcu++ (#8505)
NVIDIA/cccl: libcudacxx/test/CMakeLists.txt

      -Wno-deprecated-gpu-targets
    )
  endif()

  if (
    CCCL_ENABLE_TILE
    AND "${CMAKE_CUDA_COMPILER_VERSION}" VERSION_LESS_EQUAL "13.2"
  )
    message(
      FATAL_ERROR

>       "tile programs are not supported in NVCC until after 13.2 but found NVCC ${CMAKE_CUDA_COMPILER_VERSION}"

Reviewer: I needed 3 attempts to parse this sentence. Can we do:
```suggestion
      "tile programs are not supported in NVCC <= 13.2. NVCC version is ${CMAKE_CUDA_COMPILER_VERSION}"
```
